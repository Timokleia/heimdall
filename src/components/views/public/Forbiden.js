import React, { Component } from "react";
import PropTypes from "prop-types";
import { Container, Row } from "react-bootstrap";

class Forbiden extends Component {
  render() {
    return (
      <Container
        fluid
        style={{
          backgroundColor: "#2C3E50",
          display: "flex",
          justifyContent: "center"
        }}
      >
        <Row>
          <img src='/img/forbiden.png' style={{ width: "100%" }} />
        </Row>
      </Container>
    );
  }
}

export default Forbiden;
