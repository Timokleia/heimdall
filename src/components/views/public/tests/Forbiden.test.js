import React from "react";
import Forbiden from "../Forbiden";
import Enzyme, { mount, render, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new Adapter() });
describe("Dashboard", () => {
  it("Render without crashing", () => {
    const wrapper = shallow(<Forbiden />);
    expect(wrapper).toMatchSnapshot();
  });
});
