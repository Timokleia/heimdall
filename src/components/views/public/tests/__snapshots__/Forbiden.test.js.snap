// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Forbiden Render without crashing 1`] = `
<Container
  fluid={true}
  style={
    Object {
      "backgroundColor": "#2C3E50",
      "display": "flex",
      "justifyContent": "center",
    }
  }
>
  <Row
    noGutters={false}
  >
    <img
      src="/img/forbiden.png"
      style={
        Object {
          "width": "100%",
        }
      }
    />
  </Row>
</Container>
`;
